pkg/nodebridge/node_bridge.go

```diff
@@ -191,6 +207,29 @@ func (n *NodeBridge) listenToConfirmedMilestone(ctx context.Context, cancel cont
 	return nil
 }
 
+func (n *NodeBridge) listenToLedgerUpdates(ctx context.Context, cancel context.CancelFunc) error {
+	defer cancel()
+	stream, err := n.Client.ListenToLedgerUpdates(ctx, &inx.LedgerRequest{})
+	if err != nil {
+		return err
+	}
+	for {
+		update, err := stream.Recv()
+		if err != nil {
+			if err == io.EOF || status.Code(err) == codes.Canceled {
+				break
+			}
+			n.Logger.Errorf("listenToLedgerUpdates: %s", err.Error())
```

Comment:
Maybe use `Warnf`, so we don't print the whole stacktrace with the error message.